Casks/daisydisk.rb

```diff
@@ -3,11 +3,15 @@
   sha256 :no_check
 
   url "https://www.daisydiskapp.com/downloads/DaisyDisk.zip"
-  appcast "https://daisydiskapp.com/downloads/appcastReleaseNotes.php?appEdition=Standard&osVersion=10.15"
   name "DaisyDisk"
   desc "Disk space visualizer"
   homepage "https://daisydiskapp.com/"
 
+  livecheck do
+    url "https://daisydiskapp.com/downloads/appcastReleaseNotes.php?appEdition=Standard&osVersion=10.15"
+    regex(/Version (\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/>\s*?Version\s+?v?(\d+(?:\.\d+)+)\s*?</)
```

Anchoring this (using `>`/`<` at the start/end) will help to ensure we only match the heading text, instead of something in the description. For example, the original regex could run into a problem if the description contained something like, "Version 1.2.3 will be released in the future after more testing". If the latest version at the time was 1.2.2, then livecheck would return the nonexistent 1.2.3 version as newest.

This also loosens the regex with regard to whitespace.